pytentiostat/config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
json is a good database format but not easy for users to interact with.  I suggest we keep this (why not) but may need an easier form for the file and a little program that converts it to json?  Not sure the best.

Another possibility is yaml which is a bit easier to edit and interact as a human but just as easy to load as json (``yaml.load()``).  I am not sure the best here, just throwing out thoughts.